Utils/azuretools-core/src/com/microsoft/azuretools/utils/CommandUtils.java

```diff
@@ -113,6 +114,33 @@ private static String executeCommandAndGetOutput(final CommandLine commandLine,
         }
     }
 
+    public static ByteArrayOutputStream executeCommandAndGetOutputStream(final String command, final String[] parameters) throws IOException {
+        CommandExecutionOutput execution = executeCommandAndGetExecution(command, parameters);
+        return execution.getOutputStream();
+    }
+
+    public static DefaultExecuteResultHandler executeCommandAndGetResultHandler(final String command, final String[] parameters) throws IOException {
+        CommandExecutionOutput execution = executeCommandAndGetExecution(command, parameters);
+        return execution.getResultHandler();
+    }
+
+    public static CommandExecutionOutput executeCommandAndGetExecution(final String command, final String[] parameters) throws IOException {
+        String internalCommand = CommandUtils.isWindows() ? command + CommandUtils.COMMEND_SUFFIX_WINDOWS : command;
+        final CommandLine commandLine = new CommandLine(internalCommand);
+        commandLine.addArguments(parameters);
+        final DefaultExecuteResultHandler resultHandler = new DefaultExecuteResultHandler();
```

Comment:
final DefaultExecuteResultHandler to final DefaultExecuteResultHandler, same way for ByteArrayOutputStream DefaultExecutor , PumpStreamHandler